### PR TITLE
fix: Fixes a bug where zooming in was incorrect when hovering over overlays.

### PR DESCRIPTION
**Pull request by Savokr**
#### Type of change
Bug 

## Description :pencil:
Fixes a bug where `zoomToCursor` and `zoomPastCursor` camera modes were behaving incorrectly when cursor was hovering over html overlays that are children of `domElement` passed to `Cognite3DViewer`. 

## How has this been tested? :mag:

Hovering over loading indicator and trying to zoom in with mouse wheel in Reveal examples package. Previously it was always zooming into top left corner when hovering over any overlay, now works just fine. Also, tested with different sizes of reveal viewer that are positioned differently relative to the browser window.

## Test instructions :information_source:
Go to examples, try to zoom in hovering over loading indicator or AxisViewTool. Also, try doing the same with different `Viewer size` option in GUI. Expected behaviour is that you will zoom to the point under your cursor, not anywhere else. Also, can be tested with preview docs in HtmlOverlayTool example with removed line: `pointer-events: none;` in css style of the label.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the public documentation.
- [x] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [x] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [x] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: examples/src/pages/Viewer.tsx [hunk 1]
```diff
@@ -98,6 +98,10 @@ export function Viewer() {
         continuousModelStreaming: true,
         pointCloudEffects: {
           pointBlending: (urlParams.get('pointBlending') === 'true' ?? undefined)
+        },
```
Comment (Savokr) [inline examples/src/pages/Viewer.tsx:101]: Changed it for tests, because previously it was aligned with coordinates of domElement, so bugs similar to this one were harder to spot.

**Review by christjt — APPROVED** [review]
LGTM!

File: examples/src/pages/Viewer.tsx (hunk 1 above)
Comment (christjt) [reply examples/src/pages/Viewer.tsx:101]: Think we should keep it top left, and then deal with correctness in visual test.

**Review by christjt — APPROVED** [review]
